README.rst

```diff
@@ -82,7 +82,8 @@ chemistry and materials science, we encourage exploring
 its sister library for compiling quantum simulation algorithms in Cirq,
 `OpenFermion-Cirq <https://github.com/quantumlib/openfermion-cirq>`__.
 
-Disclaimer
+Disclaimers
 ----------
-
-Copyright 2018 The Cirq Developers. This is not an official Google product.
+* Cirq is currently in Alpha
```

Comment:
This is not enough information. We need to emphasize that we are going to be making breaking changes every single release, and e.g. that users should target specific versions if they don't want to experience unexpected breakage.